doc/tutorials/building_uos_from_clearlinux.rst

```diff
@@ -0,0 +1,160 @@
+.. _build UOS from Clearlinux:
+
+Building UOS from Clear Linux
+#############################
+
+This document builds on the :ref:`getting_started`, 
+and explains how to build UOS from Clear Linux.
+      
+Build UOS image in Clear Linux native
+*************************************
+
+In order to build out the image of UOS, 
+please follow these steps in Clear Linux native:
+
+#. In Clear Linxu native, please install ``ister`` at first, 
+   ``ister`` is a template based installer for linux, 
+   which is inclued in the bundle ``os-installer`` in Clear Linux.
+   For more information about ``ister``, 
+   please visit https://github.com/bryteise/ister.
+
+   .. code-block:: none
+
+      # sudo swupd bundle-add os-installer
```

Comment:
The shell prompt indicates you are already root, so no need to use sudo.

On a side note, I'm also thinking that we should eliminate the shell prompt in all of our code-blocks, the fact it's in there prevents people from doing a copy-paste of the instructions. @dbkinder , thoughts?